src/export_sql.py

```diff
@@ -5,28 +5,28 @@
 
 
 def export_to_sql():
-    connection = db.connect('../data/04_sql/database')
+    connection = db.connect('./data/04_sql/database')
 
-    cross_tab = pd.read_parquet('../data/03_experiment_output_aggregated/cross_tab.parquet')
+    cross_tab = pd.read_parquet('./data/03_experiment_output_aggregated/cross_tab.parquet')
```

Comment:
for me this relative path doesn't work
`FileNotFoundError: [Errno 2] No such file or directory: './data/03_experiment_output_aggregated/cross_tab.parquet'`

Please use a root variable

```python
from src.utils import root

def export_to_sql():
    connection = db.connect(root + '/data/04_sql/database')
```